features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Comments/CommentOptions.jsx

```diff
@@ -145,8 +145,17 @@ class CommentOptions extends React.Component {
         const {
             classes, comment, theme,
         } = this.props;
-        const canDelete = (comment.createdBy === AuthManager.getUser().name)
-            || (AuthManager.getUser().name === theme.custom.adminRole);
+        const user = AuthManager.getUser();
+        let username = null;
+        if (user) {
```

Comment:
We will never get to this point if a user is null, so we can safely assume that there will be a user at this point when calling`AuthManager.getUser()`

Reply:
Fixed